charter.md

```diff
@@ -53,15 +53,245 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+be held as soon as possible. [Eligible voters][voter-eligibility] from the most
+recent election will vote in the special election i.e., eligibility will not be
+redetermined at the time of the special election. A committee member elected in
+a special election will serve out the remainder of the term for the person they
+are replacing, regardless of the length of that remainder.
+
+[voter-eligibility]: /elections.md#eligibility-for-voting
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Dissolution
+
+If committee members believe that the committee is no longer able to act in
+accordance with the above [mission](#mission), the committee may dissolve by a
+few methods:
+
+* Dissolution by vote
+* Dissolution by loss of committee
+
+See [elections](#elections) for special criteria in the instance of a general
+election triggered by dissolution.
+
+#### Continuity
+
+If the committee is dissolved completely by any mechanism, selected members
+(selection described below) agree to be "continuity advisors" and attend
+meetings of the subsequently elected incoming committee in a non-voting
+capacity for 6 meetings or until the incoming committee dismisses them,
+whichever is first.
+
+##### Selection
+
+At least two (2) and not more than three (3) members shall be selected for
+this duty.
+
+* One shall be the longest-serving outgoing committee member. If multiple, the
+  incoming committee may select between them.
+* The second shall be selected by the incoming committee at their discretion.
+* The incoming committee, at their discretion, may select a third.
+
+#### Dissolution by vote
+
+Dissolution requires an affirmative vote of a **_two-thirds supermajority of
+the [fixed membership of the committee](#composition)_**, and must be unanimous
+when the committee has four (4) or fewer members.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* An affirmative vote of 5 members would be required to dissolve the committee
+
+The committee must specify a date of dissolution.
+If the committee is dissolved, all seats are vacated on the date specified.
+
+#### Dissolution by loss of committee
+
+If at any time a **_majority of the
+[fixed membership of the committee](#composition)_**'s seats are vacant, the entire committee will be dissolved.
+
+Example:
+
+* 7 (seats) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* If 4 committee seats are vacant, the entire committee will dissolve
+
+The number of vacant seats will be evaluated prior to any replacement of
+previously-seated members.
+
+### Removal
+
+A committee member may be removed from the committee by the other committee
+members. The member should be given the opportunity to resign before they are
+removed.
+
+Removal should be considered for the following reasons:
```

Comment:
I think this list is problematic and should be omitted, and a better list can be written in the future. For now I think the only reasonable requirement is that the reason be public so the community can judge the result. I think everyone is too anchored at the moment to produce a bias-free list.

Specifically, to be helpful the list here should state that: the reason given should be (or need not be) similar in severity; and also give examples of insufficient reasons. You need both positive and negative test cases to be sure your code is doing the right thing!

Reply:
While I agree that the list could use to be reviewed and built up a bit (or noted as not comprehensive), I'm also aware that there are reasons that may need to be private to avoid violations of privacy laws, for example.

Reply:
agree.  I don't think this list particularly helps

Reply:
i agree, +1 to remove the list.

Reply:
> reasons that may need to be private to avoid violations of privacy laws

In that case, the board needs to say "we have to eject member X and we can't explain the reason because it would put us in legal jeopardy".

I personally think that board members should carry liability insurance (honestly probably the CNCF should be paying for it, but if not, self-funded). And anything serious enough to kick someone off should be serious enough to defend in court.

This example has been mentioned several times in the comments here and I hate to say it, but I won't believe it unless I hear it from a lawyer. I think being a board member of an organization this size should come with reduced rights to expectation of privacy, and that crimes meriting eviction should also merit publicity. Otherwise it is too easy to abuse this system, you can do it with like only a 3 person conspiracy. 3 person conspiracies are actually possible in real life. (I'm not a lawyer and this is not a legal opinion, just a statement of what I'd want a lawyer to talk me out of to accept this reason.)

Reply:
😢 unfortunately!

Reply:
steering members agree with you re: insurance https://github.com/kubernetes/funding/issues/29 but it's not 100% complete yet

Reply:
> > reasons that may need to be private to avoid violations of privacy laws
> 
> In that case, the board needs to say "we have to eject member X and we can't explain the reason because it would put us in legal jeopardy".
> 
> I personally think that board members should carry liability insurance (honestly probably the CNCF should be paying for it, but if not, self-funded). And anything serious enough to kick someone off should be serious enough to defend in court.
> 
> This example has been mentioned several times in the comments here and I hate to say it, but I won't believe it unless I hear it from a lawyer. I think being a board member of an organization this size should come with reduced rights to expectation of privacy, and that crimes meriting eviction should also merit publicity. Otherwise it is too easy to abuse this system, you can do it with like only a 3 person conspiracy. 3 person conspiracies are actually possible in real life. (I'm not a lawyer and this is not a legal opinion, just a statement of what I'd want a lawyer to talk me out of to accept this reason.)

I can understand that. The concern as I understand it is this is a volunteer community board versus a paid organization. While I prefer publication of reasons of eviction both for the community's health as well as for an assurance that no one is taking advantage of the community, my (fairly) limited experience with volunteer boards is there is a concern around privacy laws, and often the generic wording of separation is the choice. Is it ideal? No, not to me. But it's definitely something that we have to think about (and I'm also not a lawyer! So take my thoughts with a grain of salt considering I haven't had enough tea yet today...). In the end, the steering committee answers to and serves the community, and I agree that, where possible, providing reasons are healthy for the community itself.

Reply:
@nimbinatus I heard "asymmetric lawyering capabilities", therefore "avoid nuisance lawsuits", and I can definitely appreciate that. But I highly doubt omitting a reason will achieve that goal, and it might even make such a lawsuit more likely! It might make it worse if it creates the appearance that the board doesn't believe it is doing something legally defensible!

Reply:
> @nimbinatus I heard "asymmetric lawyering capabilities", therefore "avoid nuisance lawsuits", and I can definitely appreciate that. But I highly doubt omitting a reason will achieve that goal, and it might even make such a lawsuit more likely! It might make it worse if it creates the appearance that the board doesn't believe it is doing something legally defensible!

Very fair point. All the more reason to ensure a lawyer reviews the comms if we have that availability (would the CNCF pay for this? Do we have someone on retainer? I would expect that to be true.), and all the more reason to have a clear documentation guide for explicit removals versus people voluntarily stepping down. I'm assuming that documentation guide is elsewhere. In other volunteer organizations I've been part of, there have been cases where revealing a reason was determined to likely lead to revealing identities and/or retaliation against a victim, reporter, or a voting board member. Therefore the reason the vote was called and the vote itself (and the discussion) was documented and sealed as proof against a lawsuit while maintaining privacy and safety of others, and community members were given a generic note. The concern comes from my own experience on those kinds of boards and organizations; again, that experience is limited, not comprehensive, so it's my two cents. I trust that the steering committee is considering both of our feedback when reviewing these docs.

Reply:
I agree it's a terrible dilemma to choose between protecting a 3rd party and demonstrating sufficiently skeptical truth acquisition ability. Fortunately I think these rules don't have to take a position on this right now.